qa/1543.out

```diff
@@ -1031,6 +1031,23 @@ sample_control{role="testing",agent="sample",hostname="HOSTNAME",cluster="zero",
 done full scrape
 == check pmproxy is running ==
 pmproxy check passed
+== good filter globbing ==
+# PCP5 sample.long.one 29.0.10 32 PM_INDOM_NULL instant none
+# HELP sample_long_one 1 as a 32-bit integer
+# TYPE sample_long_one gauge
+sample_long_one{role="testing",agent="sample",hostname="HOSTNAME",cluster="zero",domainname="DOMAINNAME",machineid="MACHINEID"} 1
+== good filter exact match ==
+# PCP5 sample.long.one 29.0.10 32 PM_INDOM_NULL instant none
+# HELP sample_long_one 1 as a 32-bit integer
+# TYPE sample_long_one gauge
+sample_long_one{role="testing",agent="sample",hostname="HOSTNAME",cluster="zero",domainname="DOMAINNAME",machineid="MACHINEID"} 1
+== good filter regex ==
+# PCP5 sample.long.one 29.0.10 32 PM_INDOM_NULL instant none
+# HELP sample_long_one 1 as a 32-bit integer
+# TYPE sample_long_one gauge
+sample_long_one{role="testing",agent="sample",hostname="HOSTNAME",cluster="zero",domainname="DOMAINNAME",machineid="MACHINEID"} 1
+== bad filter match param ==
+{"context":562009464,"message":"dummy - invalid 'match' parameter value","success":false}

```

Comment:
We'll need to add a sed filter on the context number here @wcohen (in test script qa/1543).